act-rules-format.bs

```diff
@@ -321,17 +325,33 @@ Each new release of an ACT Rule MUST be identifiable with either a date or a ver
 </div>
 
 
-Issues List {#updates-issues}
--------------------------------------
+Issues List (optional) {#issues-list}
+=====================================
 
-An ACT Rule MAY include an issues list. When included, the issues list MUST be used to record cases when the [=outcome=] of an ACT Rule was `failed` where it ought to have been `passed`, or vice versa. There are several reasons why this might occur, including:
+An ACT Rule MUST include a list or a reference to a list of any known issues. The issues list MUST be used to record cases where the [=outcome=] of an ACT Rule was `failed` where it ought to have been `passed` or `inapplicable`, or vice versa. There are several reasons why this might occur, including:
```

Comment:
How can this section be marked as optional yet have a MUST requirement?

Reply:
Good question! Definitely needed a cleanup.
Now it is MUST + not optional.
However, MAY + optional is also an option. 
Question is: if there are known issues, would it be okay not to list them in the ACT Rule?